run_scripts/2_fetch_raw_data.py

```diff
@@ -0,0 +1,93 @@
+import argparse
+import json
+import os
+import subprocess
+import tempfile
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description="Fetches all the raw data for this project from Rapid Pro")
+
+    parser.add_argument("user", help="Identifier of the user launching this program")
+    parser.add_argument("pipeline_configuration_file_path", metavar="pipeline-configuration-file",
+                        help="Path to the pipeline configuration json file")
+    parser.add_argument("rapid_pro_tools_dir", metavar="rapid-pro-tools-dir",
+                        help="Path to a directory to checkout the required version of RapidProTools to")
+    parser.add_argument("root_data_dir", metavar="root-data-dir",
+                        help="Path to the root of the project data directory")
+
+    args = parser.parse_args()
+
+    user = args.user
+    pipeline_configuration_file_path = args.pipeline_configuration_file_path
+    rapid_pro_tools_dir = args.rapid_pro_tools_dir
+    root_data_dir = args.root_data_dir
+
+    uuid_table_path = f"{root_data_dir}/UUIDs/phone_uuids.json"
+
+    SHOWS = [
+        "csap_s01e01_activation",
+        "csap_s01e02_activation",
+        "csap_s01e03_activation",
+        "csap_s01e04_activation"
+    ]
+
+    SURVEYS = [
+        "csap_demog",
+        "csap_evaluation"
+    ]
+
+    TEST_CONTACTS_PATH = os.path.abspath("./test_contacts.json")
+
+    # Read the settings from the configuration file
+    with open(pipeline_configuration_file_path) as f:
+        pipeline_config = json.load(f)
+
+        rapid_pro_base_url = pipeline_config["RapidProBaseURL"]
+        rapid_pro_token_file_url = pipeline_config["RapidProTokenFileURL"]
+
+    # Download/checkout the appropriate version of RapidProTools
+    exit_code = subprocess.call(["./checkout_rapid_pro_tools.sh", rapid_pro_tools_dir])
+    assert exit_code == 0
```

Comment:
Please add a diagnostics log line here - I would suggest adding a logging function that at the moment just prefixes the log with a DateTime stamp and forwards it to print - but will also give us a hook for doing better thing

Reply:
added error strings to the asserts